src/scripts/browserDeviceProfile.js

```diff
@@ -1263,17 +1267,18 @@ export default function (options) {
         // This paired with the "Prefer fMP4-HLS Container" client playback setting enables DOVI playback on webOS.
         profile.CodecProfiles.push({
             Type: 'Video',
-            Container: '-mp4',
+            Container: 'ts,mp4',
             Codec: 'hevc',
             Conditions: [
                 {
                     Condition: 'EqualsAny',
                     Property: 'VideoRangeType',
-                    Value: 'SDR|HDR10|HLG',
+                    Value: hevcVideoRangeTypes,
                     IsRequired: false
                 }
             ]
         });
+        hevcCodecProfileConditions[2].Value = hevcVideoRangeTypes.split('|').filter((v) => !v.startsWith('DOVI')).join('|');
```

Comment:
I think accessing the condition via an index is unreliable.
Maybe something like:
```js
hevcCodecProfileConditions.find(x => x.Property === 'VideoRangeType')?.Value = ...
```
_Not tested._